infra/gcp/bash/ensure-prod-storage.sh

```diff
@@ -87,7 +89,9 @@ readonly PROD_PROJECT_DISABLED_SERVICES=(
 )
 
 # Regions for prod GCR.
-PROD_REGIONS=(us eu asia)
+GCR_PROD_REGIONS=(us eu asia)
+# Regions for prod AR.
+AR_PROD_REGIONS=(asia-east1 asia-northeast1 europe-north1 europe-west1 europe-west4 us-central1 us-east1 us-east4)
```

Comment:
@puerco is it ok to push around 15 container registries during image promotion ?

Reply:
In principle there should be no problem. Promoting to that many registries will increase the promotion time so we should test to know how much more to ensure we don't timeout if it is a lot (I don't expect it to, but still).